### PR TITLE
[Fix #3] Show elapsed time instead of time left

### LINKED ISSUE #3: Time up plutôt que time down

### PR DESCRIPTION
**Pull request by gregoirenovel**

### CODE REVIEW

File: app/components/standup-deck.js [hunk 1]
```diff
@@ -35,19 +35,20 @@ export default Ember.Component.extend({
 
   setTimerForState: function(state) {
     var seconds = state === 'startups' ? 65 : 315,
+        starttime = Date.parse(new Date()),
```
Comment (maukoquiroga) [inline app/components/standup-deck.js]: `startTime` ?

**Comment by gregoirenovel** [comment]
@maukoquiroga done

**Comment by maukoquiroga** [comment]
Independent of the requested change, two things I noticed today:

1. We win by showing people how much they've talked already, but we lose a way to tell them their time is finishing. Do you have any ideas of how we could solve that? Fading the timer's colour ? Like 🍏 => 💛 => 🔴 ?

2. And the second thing is seconds now go until 1:04, but it should be clearer if the start at -0:04 maybe? Perhaps 1. will solve this as wel...

**Comment by gregoirenovel** [comment]
> 1. We win by showing people how much they've talked already, but we lose a way to tell them their time is finishing. Do you have any ideas of how we could solve that? Fading the timer's colour ? Like 🍏 => 💛 => 🔴 ?

We could indeed make the timer red for the last 15 seconds.

> 2. And the second thing is seconds now go until 1:04, but it should be clearer if they start at -0:04 maybe? Perhaps 1. will solve this as well...

Hmmm, I'm not sure about this one, I fear that the negative values might confuse some people. Maybe for the first 5 seconds we could show a "spinner" (with dots or something). But I'm not certain how big of an issue this is.

**Comment by maukoquiroga** [comment]
@gregoirenovel I'm sorry I edited my comment but I shouldn't have. Forget about 2.

**Comment by gregoirenovel** [comment]
@maukoquiroga du coup c'est OK pour merge ?

**Comment by gregoirenovel** [comment]
@maukoquiroga à jour

**Pull request merged**